pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
when building or releasing the gem, it load he gemspec file as ruby without any dependency and try to load version.rb to get the version number. so it parse :
```ruby
class Sysfs < Driver
  VERSION = ...
end
```
if Driver is not already declared you have an error.